recipes-aws/python/python-gg-group-setup.inc

```diff
@@ -0,0 +1,27 @@
+SUMMARY = "A file-driven approach to the creation of an entire AWS Greengrass group"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=e3fc50a88d0a364313df4b21ef20c29e"
+
+PYPI_PACKAGE = "gg_group_setup"
+
+SRC_URI[md5sum] = "afc38bf424421634bea65fe32f2a32dd"
+SRC_URI[sha256sum] = "5d68dc5007a1bf24e8d1415db93e4f2b4dde42ffd98b868778bb3f6522d4d89d"
+
+DEPENDS += " \
+    ${PYTHON_PN}-boto3 \
+    ${PYTHON_PN}-fire \
+"
```

Comment:
Is this necessary for the package to build or are runtime dependencies enough?

Reply:
That'll teach me to push things upstream I did ages ago without checking properly! The RDEPENDS are fine.